tests/suites/test_suite_ccm.data

```diff
@@ -1554,30 +1554,94 @@ CCM encrypt, overflow ad NIST VADT AES-192 #14 (P=24, N=13, A=13, T=16)
 depends_on:MBEDTLS_AES_C
 mbedtls_ccm_overflow_ad:MBEDTLS_CIPHER_ID_AES:MBEDTLS_CCM_ENCRYPT:"b5f43f3ae38a6165f0f990abe9ee50cd9ad7e847a0a51731":"13501aebda19a9bf1b5ffaa42a":"ead4c45ff9db54f9902a6de181"
 
+CCM encrypt, incomplete ad NIST VADT AES-192 #14 (P=24, N=13, A=13, T=16)
```

Comment:
Please change the corner case and instant finish test cases to using AES-128. Some platforms lack AES-192, but they should still be able to run the corner case tests.